include/eve/module/core/regular/impl/simd/x86/minmax.hpp

```diff
@@ -0,0 +1,43 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/concept/value.hpp>
+#include <eve/detail/apply_over.hpp>
+#include <eve/detail/category.hpp>
+#include <eve/detail/implementation.hpp>
+
+namespace eve::detail
+{
+// -----  Regular
+template<arithmetic_scalar_value T, typename N>
+EVE_FORCEINLINE auto
+minmax_(EVE_SUPPORTS(sse2_), wide<T, N> v0, wide<T, N> v1) noexcept requires x86_abi<abi_t<T, N>>
```

Comment:
Why not just cpu.

Reply:
We usually let CPU be the full non-SIMD case.